src/server.py

```diff
@@ -52,6 +63,14 @@ def html_to_text(html):
     except Exception:
         return ""
 
+def replace_title_text_from_title_url(article):
+    urls = re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+', article.title)
+    if len(urls)> 0:
+        article_from_url = get_article(urls[0], Config())
```

Comment:
instead of passing Config() python allows to have default value for the function arguments. 
You can write something like: `def get_article(url, config =  Config())`

Reply:
fixed